src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -7715,7 +7743,7 @@ namespace OutputReportTabular {
             // determine which resource is the primary heating resourse
             resourcePrimaryHeating = 0;
             heatingMaximum = 0.0;
-            for (iResource = 1; iResource <= 5; ++iResource) { // don't do water
+            for (iResource = 1; iResource <= 12; ++iResource) { // don't do water
```

Comment:
Changed upper limit in FOR loop to 12 here.

Reply:
It's because this FOR loop does not count "water" resource. Some FOR loops that count water resource use 13, like below.